Source/Scene/MetadataSemantic.js

```diff
@@ -57,6 +57,86 @@ var MetadataSemantic = {
    * @private
```

Comment:
(Above the fold) - it looks like we had constants for `MINIMUM_HEIGHT` and `MAXIMUM_HEIGHT`, which are not in the current [Semantics spec](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata/Semantics). 

Should we remove these two older constants or does something need them?

Reply:
Yes you can remove anything that's outdated